src/styles/components.css

```diff
@@ -73,7 +73,7 @@
   min-height: 24px;
 }
 
-.Select-control:hover {
+.css-13cymwt-control:hover {
```

Comment:
What is this strange class ?

Reply:
fixed in https://github.com/kbss-cvut/s-forms-components/pull/27/commits/4aa64f8cd10479cd7279faf6c14fe22365be6599